pipenv/environments.py

```diff
@@ -24,6 +24,36 @@ def _is_env_truthy(name):
     return os.environ.get(name).lower() not in ("0", "false", "no", "off")
 
 
+def get_from_env(arg, prefix="PIPENV", check_for_negation=True):
+    """
+    Check the environment for a variable, returning its truthy or stringified value
+
+    For example, setting ``PIPENV_NO_RESOLVE_VCS=1`` would mean that
+    ``get_from_env("RESOLVE_VCS", prefix="PIPENV")`` would return ``False``.
+
+    :param str arg: The name of the variable to look for
+    :param str prefix: The prefix to attach to the variable, defaults to "PIPENV"
+    :param bool check_for_negation: Whether to check for ``<PREFIX>_NO_<arg>``, defaults
+        to True
+    :return: The value from the environment if available
+    :rtype: Optional[Union[str, bool]]
+    """
+    negative_lookup = "NO_{0}".format(arg)
+    positive_lookup = arg
+    if prefix:
+        positive_lookup = "{0}_{1}".format(prefix, arg)
+        negative_lookup = "{0}_{1}".format(prefix, negative_lookup)
+    if positive_lookup in os.environ:
+        if _is_env_truthy(positive_lookup):
+            return bool(os.environ[positive_lookup])
+        return os.environ[positive_lookup]
```

Comment:
It seems wrong to me. Think of `ENABLE_SOMETHING=0`, does it mean to enable or disable?

Reply:
so the answer to that is it should be disabled -- the real answer is that I was not at all thinking about how `_is_env_truthy` works and it really can't be used here to achieve this

Reply:
Yeah, you are probably seeking for `_is_env_boolean_like`.

Reply:
oh gosh is that a thing, I couldn't think of the name of it and probably just reinvented it

Reply:
It is not implemented, I just make up the name that indicates how the function should work.